src/win_hv_emulation.rs

```diff
@@ -21,34 +21,32 @@ use win_hv_platform_defs::*;
 pub type WHV_EMULATOR_IO_PORT_CALLBACK =
     extern "stdcall" fn(Context: *mut VOID, IoAccess: *mut WHV_EMULATOR_IO_ACCESS_INFO) -> HRESULT;
 
-pub type WHV_EMULATOR_MEMORY_CALLBACK =
-    extern "stdcall" fn(Context: *mut VOID, MemoryAccess: *mut WHV_EMULATOR_MEMORY_ACCESS_INFO)
-        -> HRESULT;
-
-pub type WHV_EMULATOR_GET_VIRTUAL_PROCESSOR_REGISTERS_CALLBACK =
-    extern "stdcall" fn(
-        Context: *mut VOID,
-        RegisterNames: *const WHV_REGISTER_NAME,
-        RegisterCount: UINT32,
-        RegisterValues: *mut WHV_REGISTER_VALUE,
-    ) -> HRESULT;
-
-pub type WHV_EMULATOR_SET_VIRTUAL_PROCESSOR_REGISTERS_CALLBACK =
-    extern "stdcall" fn(
-        Context: *mut VOID,
-        RegisterNames: *const WHV_REGISTER_NAME,
-        RegisterCount: UINT32,
-        RegisterValues: *const WHV_REGISTER_VALUE,
-    ) -> HRESULT;
-
-pub type WHV_EMULATOR_TRANSLATE_GVA_PAGE_CALLBACK =
-    extern "stdcall" fn(
-        Context: *mut VOID,
-        Gva: WHV_GUEST_VIRTUAL_ADDRESS,
-        TranslateFlags: WHV_TRANSLATE_GVA_FLAGS,
-        TranslationResult: *mut WHV_TRANSLATE_GVA_RESULT_CODE,
-        Gpa: *mut WHV_GUEST_PHYSICAL_ADDRESS,
-    ) -> HRESULT;
+pub type WHV_EMULATOR_MEMORY_CALLBACK = extern "stdcall" fn(
```

Comment:
Those changes seem unrelated. I think it's been a while since "rustfmt" has been run against this file, so perhaps its rules have changed meanwhile.

I suggest sending separate commits for code style changes, otherwise patches get larger and more difficult to review.

Reply:
Agreed, a separate commit containing only the rustfmt changes would have been better. Would you like me to break them out for this PR, or is this a suggestion for future PRs?

Reply:
However works best for you, I think it's not worth blocking the patch just for that.

If you do wish to move those to a separate patch, one idea would be to just go back one commit, do a "cargo fmt", commit the changes and then apply back the patch. It should be a one minute thing, although you may get a few conflicts.

Reply:
This was a good suggestion--the separate commits are much clearer now.

Reply:
This was a good suggestion--the commits are much clearer now.